packages/extension-ui/src/components/EditMenuCard.tsx

```diff
@@ -176,7 +176,7 @@ export default styled(EditMenuCard)(
 
   .flex-group {
     display: flex;
-    gap: 4px;
+    gap: 40px;
```

Comment:
it's not trivial in css, but someone wrote address ellipsis in js :sweat_smile:

packages/extension-ui/src/components/Address.tsx:247
packages/extension-ui/src/util/ellipsisName.ts

I don't mind this solution, but I don't think it's that much work to make those two match

Reply:
My bad, I was looking at the address, but the issue concerns `Name` :disappointed:  :palm_tree: 
I checked the name styles:
```
  .name {
    font-size: 16px;
    line-height: 125%;
    letter-spacing: 0.06em;
    font-family: ${theme.secondaryFontFamily};
    font-weight: 500;
    margin: 2px 0;
    overflow: hidden;
    text-overflow: ellipsis;
    width: 300px;
    white-space: nowrap;
    color: ${theme.textColor};
    width: 190px;
    margin-right: 4px;
  }
```
copying those here should work, right?
Anyway, it's probably not a deal breaker :sweat_smile:

Reply:
Hmm, not sure I get it - why would I want to copy those styles?

Reply:
sorry, I was referencing this discussion about number of letters:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/53d00a50-293a-48dc-9968-729300a71562)

Reply:
Ok, I get it. But the ultimate goal is to fill the unused space, I believe. I don't feel comfortable trying to reach the same lengths between differently styled components, because that's heursitic hacks against the ellipssis mechanism, so chasing such a goal is delusional and will inevitalby lead to false positives during desing<->implementation matching audits. As in: no user will be counting characters to see if they match between the views, but if we change some style somewhere and it causes over/under-flow, a user will notice immediately. So with the design requirements, we should work with the flow of how elipssis works in css, not against it.

I sort of was trying to convey that in the jira comments, not sure if I managed 😄

Reply:
We didn't need to wait long time for a situation in which trying to achieve the exact same characters count would break layout :D (well, it's actually about constant width, not constant chars length, but it's the exact same problem):
They've hardcoded width of an element (which btw just begs to be a flex-grow item instead):

![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/6209244/aa8ec76f-e2be-4b5c-bbb5-bf98805a0eaf)

This looked all right when in the extension view, but is broken in the popup view in which the gray outter borders are added to the window, lowering the room for any interiors, thus preventing the entire list item from fitting in the available space. If the content was flex-grow (or correspondingly: if the text was a non-const-char-length ellipssis) it would adapt and fit.